### ISSUE TITLE
Adding a country to allowed countries list fails

### ISSUE THREAD

**Issue by daboss84** [OpenMage/magento-lts#2601]
### Preconditions (*)

1. OM 20.0.16 (but problem exists since it was Magento CE)
2. PHP 7.4

### Steps to reproduce (*)

1. Go to System | Configuration | Common (? "Allgemein") -> Country options -> Allow countries
2. Select one or more additional countries
3. Save

### Expected result (*)

1. Saving succeeds

### Actual result (*)

1. Saving failes
2. See screenshot (sorry, in German) 
![grafik](https://user-images.githubusercontent.com/18015530/190230621-bd9360af-8adb-4aeb-ab73-97fe113e9dbc.png)

### Possible solution
`Mage_Directory_Model_Resource_Region_Collection::__construct()`
Remove line
`$this->addOrder('name', Varien_Data_Collection::SORT_ORDER_ASC);`

**Comment by fballiano** [comment]
can't reproduce the problem, can you select english from the combobox on the lower left corner and repost the screenshot en english? i'm not if I'm in the right section (but I think I am), althugh I can save with no error

**Comment by sreichel** [comment]
I see you use an outdated module (Market Ready Germany), maybe its caused by this.

Please disable it and try again.

Btw ... better use https://github.com/firegento/firegento-magesetup

**Comment by daboss84** [comment]
Here is the same screenshot in english:
![grafik](https://user-images.githubusercontent.com/18015530/190233575-dc9db22e-7698-4db0-bf38-2bf19715d1c3.png)

**Comment by fballiano** [comment]
ok so I was in the right section but no errors

**Comment by sreichel** [comment]
MRG changed a lot of things. I cant check b/c i cant find any downloads for it :(

**Comment by sreichel** [comment]
Into the blue .... this could be related to `Symmetrics_TweaksGerman_Model_System_Config_Backend_Region::_beforeSave()`.

**Comment by ADDISON74** [comment]
I was not able to replicate this issue. I used as test environment DDEV, OpenMage v19 cloned a few hours ago, Magento Sample Pack.

**Step 1** - I clicked the [Save Config] button without doing anything on the page. The green message appears.

![step1](https://user-images.githubusercontent.com/8360474/190262702-e432a6b5-75ce-46da-8bd7-463c824b87ed.png)

**Step 2** - I deselected all countries then clicked on the [Save Config] button. The green message appears.

![step2](https://user-images.githubusercontent.com/8360474/190262720-60bba6ec-8903-43bf-ac62-869036a5ef57.png)

**Step 3** - I selected all countries then clicked on the [Save Config] button. The green message appears.

![step3](https://user-images.githubusercontent.com/8360474/190262751-7490b2a7-e443-451f-8b6d-98de6ca1940e.png)

As can be seen from the screenshot posted by the author, we are discussing about an OpenMage version that is not stock, but with extensions and modifications, probably a production installation.

@daboss84 - I recommend you to go to ```/app/etc/modules``` and disable one by one the used modules, other than those with the **Mage_** prefix and check what happens after each change in the files. Once you have identified the extension, search in its directory for the file that contains the _beforeSave() method.

**Comment by daboss84** [comment]
Well well well... I also checked on a clean install and it worked there.

I then deactivated lots of add ons via /app/etc/*.xml (not Symmetrics stuff, though) - still broken.
Following @sreichel's guess, I commented out _beforeSave to just return this and do nothing else - then it works.

So my next steps would be to reactivate all other plugins. If it still works, I'll dig more into the _beforeSave (But for now, unfortunately, I have to work on other topics)

So thank you so far.
